src/main/java/internet/shop/controller/order/GetUserOrders.java

```diff
@@ -0,0 +1,26 @@
+package internet.shop.controller.order;
+
+import internet.shop.lib.Injector;
+import internet.shop.model.Order;
+import internet.shop.service.OrderService;
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class GetUserOrders extends HttpServlet {
```

Comment:
GetUserOrders -> GetUserOrdersController ?

Reply:
I missed this moment here. Thank's!